stader-cli/node/utilize-sd.go

```diff
@@ -85,7 +84,8 @@ func utilizeSD(c *cli.Context) error {
 }
 
 func GetMinUtility(sdStatus *api.SdStatusResponse) *big.Int {
-	minUtility := new(big.Int).Sub(sdStatus.SdCollateralRequireAmount, sdStatus.SdCollateralCurrentAmount)
+	totalCollateral := new(big.Int).Sub(sdStatus.SdUtilizerLatestBalance, sdStatus.SdCollateralCurrentAmount)
```

Comment:
This should be `Add`. Its `SdCollateralRequireAmount - (SdUtilizerBalance + SdCollateralCurrentAmount)`